pkg/cluster/member.go

```diff
@@ -34,7 +34,7 @@ func (c *Cluster) updateMembers(known etcdutil.MemberSet) error {
 		if c.cluster.Spec.SelfHosted != nil {
 			name = m.Name
 			if len(name) == 0 || len(m.ClientURLs) == 0 {
-				c.logger.Errorf("member peerURL (%s): %v", m.PeerURLs[0], errUnexpectedUnreadyMember)
+				c.logger.Warningf("member peerURL (%s): %v", m.PeerURLs[0], errUnexpectedUnreadyMember)
```

Comment:
I think this and https://github.com/coreos/etcd-operator/pull/1177#discussion_r121023105 are the same thing.

It's confusing to users to see such "error" logs while it shouldn't.